vnext/ReactWindowsCore/AsyncStorage/AsyncStorageManager.h

```diff
@@ -38,15 +37,13 @@ class AsyncStorageManager {
     xplat::module::CxxModule::Callback m_jsCallback;
   };
 
- private:
```

Comment:
>private: [](start = 1, length = 8)

Please keep it because it is here to logically separate methods and fields.

Reply:
is there a URL for this part of the coding standard so I can read up on this rule and the other expectations and better follow them?

Reply:
@matthargett , Are you running some kind of automated tool for these changes .. a bunch of these changes look like it.

I don't know that this is a hard req from our side.  I suspect it was a preference.

Reply:
Not running an automatic tool, I'm just following warnings around and eyeballing files. Using duplicate accessibility section declarations versus regions or comments is an interesting preference. If it needs to be carried forward, I can add this some of the other implicit style rules to the contribution document.